roles/etcd/tasks/update_cas.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Create a temporary directory
+  tempfile:
+    state: directory
+    suffix: csr_etcd
+  register: csr_staging_dir_mktemp
+
+- name: Unzip the certificate tarball
+  unarchive:
+    src: "{{ openshift_csr_issued_certificates }}"
+    dest: "{{ csr_staging_dir_mktemp.path }}"
+
+# Create a bundle to link ca.crt to for etcd. The bundle contains the old and new CAs to ease rollover
+- name: Check for ca-bundle.crt
+  stat:
+    path: "{{ etcd_cert_config_dir }}/ca-bundle.crt"
+  register: ca_bundle_stat
+  failed_when: false
+
+- name: Check for current ca.crt
+  stat:
+    path: "{{ etcd_cert_config_dir }}/ca.crt"
+  register: ca_crt_stat
+  failed_when: false
+
+- name: Check for staged ca.crt
+  stat:
+    path: "{{ csr_staging_dir_mktemp.path }}/etcd-ca/ca.crt"
+  register: staged_ca_crt_stat
+  failed_when: false
+
+- name: Cat staged ca.crt
+  command: "cat {{ csr_staging_dir_mktemp.path }}/etcd-ca/ca.crt"
+  register: ca_cat
+  when: staged_ca_crt_stat.stat.exists
+
+- name: Migrate ca.crt to ca-bundle.crt
+  command: mv ca.crt ca-bundle.crt
+  args:
+    chdir: "{{ etcd_cert_config_dir }}"
+  when: ca_crt_stat.stat.isreg and not ca_bundle_stat.stat.exists
+
+- name: Append staged CA to bundle
+  lineinfile:
+    path: "{{ etcd_cert_config_dir }}/ca-bundle.crt"
+    line: "{{ ca_cat.stdout }}"
+  when: staged_ca_crt_stat.stat.exists
+
+- name: Link ca.crt to ca-bundle.crt
+  file:
+    src: "{{ etcd_cert_config_dir }}/ca-bundle.crt"
+    path: "{{ etcd_cert_config_dir }}/ca.crt"
```

Comment:
@smarterclayton Given our discussion on client-ca.crt, all of these ca.crt (and elsewhere) should be updated to client-ca.crt, correct?  And should we be symlinking to the bundle or just replacing client-ca.crt with the bundle?  Will having this bundle this affect how the kubelet manages the rotation for locally signed certs?